src/pages/Pool/PoolPosition/PoolPosition.less

```diff
@@ -0,0 +1,52 @@
+.position__detail_wrapper {
+  width: 420px;
+  background-color: var(--ergo-pool-position-bg-1);
+}
+
+.percent-lbl.ant-typography {
+  padding: 2px 4px;
+  margin-left: 8px;
+  border-radius: 8px;
+  font-size: 12px;
+  &.token-pair {
+    background-color: var(--ergo-pool-position-bg);
+  }
+  &.token {
+    background-color: var(--ergo-pool-position-bg-1);
+  }
+}
+
+.collect-fee__btn.ant-btn.ant-btn-primary {
```

Comment:
I added 8px border-radius to the `Button` component when it has `size=small`
Remove this styles, we will not use collect-fee button in v1 anyway